### PR TITLE
Code Review Fixes 

### PR DESCRIPTION
**Pull request by shandilya3**
## Description

Addressing code review comments on the feature branch [In browser messages](https://github.com/adobe/alloy/pull/1029#top) PR : https://github.com/adobe/alloy/pull/1029

Code Review based fixes Includes 
- [X] Using Alloy Utils
- [X] Constant used in multiple components moved to shared module
- [X]  Added try catch 
- [X] removed __adobe>ajo>in-app-response-format
- [X] few minor fixes

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [X] I have made any necessary test changes and all tests pass.
- [X] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/DecisioningEngine/createEvaluableRulesetPayload.js [hunk 1]
```diff
@@ -31,14 +31,20 @@ const isRulesetItem = item => {
...
+        ? JSON.parse(data.content)
+        : data.content;
+
+    return (
+      content &&
+      Object.prototype.hasOwnProperty.call(content, "version") &&
+      Object.prototype.hasOwnProperty.call(content, "rules")
+    );
+  } catch (error) {
+    return false;
```
Comment (shandilya3) [inline src/components/DecisioningEngine/createEvaluableRulesetPayload.js:46]: Do we also want to log a message here, or can we live without it?

File: src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js [hunk 2]
```diff
@@ -11,23 +11,33 @@ governing permissions and limitations under the License.
...
 
 const ALLOY_MESSAGING_CONTAINER_ID = "alloy-messaging-container";
 const ALLOY_OVERLAY_CONTAINER_ID = "alloy-overlay-container";
 const ALLOY_IFRAME_ID = "alloy-content-iframe";
 
-const dismissMessage = () =>
-  [ALLOY_MESSAGING_CONTAINER_ID, ALLOY_OVERLAY_CONTAINER_ID].forEach(
-    removeElementById
-  );
+export const dismissMessage = () => {
```
Comment (jasonwaters) [inline src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js:26]: What is the reason you removed the `removeElementById` method?  I find the prior code for `dismissMessage` to be more readable.  Could we keep that method but modify it to use `removeNode`?

**Review by jasonwaters — CHANGES_REQUESTED** [review]
Looks good, just one question/request

File: src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js (hunk 2 above)
Comment (shandilya3) [reply src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js:26]: I believe this code is quite readable. The `removeElementById` function might seem a bit redundant, especially when we already have `removeNode`. However, I don't have a strong preference, so I go ahead and include `removeElementById` back in the code.

**Review by jasonwaters — APPROVED** [review]

**Pull request merged**